### DISCUSSION TITLE
ch04-05의 파일이 열리지 않습니다.

### DISCUSSION THREAD

**Discussion by Bangseungjae** [yoonjeong-kwon/fastcampus-kubernetes#12] [Q&A] [answered]
혹시 tmdwo5825@naver.com 으로 강의자료를 받을 수 있을까요?

**Comment by yoonjeong-kwon** [comment, marked as answer]
안녕하세요! 요청하신 메일로 전송했습니다.
감사합니다. :)

**Reply by yunjaekim00** [reply]
안녕하세요. 저도 파일이 안 열리네요. yunjaekim007@gmail.com 로 자료 부탁드립니다. 감사합니다~

**Reply by yunjaekim00** [reply]
아... 해결했어요. 윈도우에서 7zip으로는 안 풀렸는데 powershell 에서 Expand-Archive 명령어를 쓰니 풀리네요.
